jobs/V3/auto_complete_sample.rb

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def job_discovery_job_title_auto_complete company_name:, query:, project_id:
+  # [START job_discovery_job_title_auto_complete]
+  # company_name  = "The company's name which the job belongs to"
+  # query         = "Job title prefix as auto complete query"
```

Comment:
Use the wording from the API reference docs for attributes. 
For example `query`: The query string that matches against the job title, description, and location fields.

https://cloud.google.com/talent-solution/job-search/docs/reference/rest/v3/JobQuery